scheds/rust/scx_rusty/src/bpf/intf.h

```diff
@@ -87,9 +93,17 @@ enum stat_idx {
 	RUSTY_STAT_DL_CLAMP,
 	RUSTY_STAT_DL_PRESET,
 
+	RUSTY_STAT_IOWAIT_BOOST,
+
 	RUSTY_NR_STATS,
 };
 
+struct boost_ctx {
+	u64		deadline;
+	unsigned int	level;
```

Comment:
This field really isn't used properly right now, ideally there would be some decay based on the deadline or something. For now it just gets set to the max and sort of works.

Reply:
Makes sense. Just to keep the code a bit cleaner and less complex in the interim, are you OK with removing `level`, and then re-adding it when we have a concrete idea we want to try out?